package.json

```
@@ -0,0 +1,20 @@
+{
+  "name": "mq-events",
```

Looking over the code again, I realized that we should probably use a name with an org included (and grab the org name on npm), especially since a package called [`mq-events`](https://www.npmjs.com/package/mq-events) already exists.

```suggestion
  "name": "@esamarathon/mq-events",
```

---

Seems like a good idea, I only put the name in there as a "placeholder" anyway.